app/api/rest_api.py

```diff
@@ -88,7 +88,8 @@ def __init__(self, services):
                     contact=lambda d: self.rest_svc.download_contact_report(d),
                     configuration=lambda d: self.rest_svc.update_config(d),
                     link=lambda d: self.rest_svc.get_potential_links(**d),
-                    operation=lambda d: self.rest_svc.update_operation(**d)
+                    operation=lambda d: self.rest_svc.update_operation(**d),
+                    chain=lambda d: self.rest_svc.find_link(**d)
```

Comment:
i don't think we need to add an extra endpoint to return the extra_info block. since all links are on the operation object, we should be able to use just that.